gateway-service/src/main/java/com/ca/mfaas/gateway/security/service/JwtSecurityInitializer.java

```diff
@@ -10,6 +10,8 @@
 
 package com.ca.mfaas.gateway.security.service;
 
+import com.ca.mfaas.message.log.ApimlLogger;
+import com.ca.mfaas.product.logging.annotations.InjectApimlLogger;
 import com.ca.mfaas.security.HttpsConfig;
 import com.ca.mfaas.security.HttpsConfigError;
 import com.ca.mfaas.security.SecurityUtils;
```

Comment:
Here is too... Redundant  Slf4j annotation.

Reply:
Fixed this one too